setup.py

```diff
@@ -101,6 +101,12 @@
             "default = qiskit.transpiler.passes.synthesis.unitary_synthesis:DefaultUnitarySynthesis",
             "aqc = qiskit.transpiler.synthesis.aqc.aqc_plugin:AQCSynthesisPlugin",
         ],
+        "qiskit.synthesis": [
+            "clifford.default = qiskit.transpiler.passes.synthesis.high_level_synthesis:DefaultSynthesisClifford",
+            "linear_function.default = qiskit.transpiler.passes.synthesis.high_level_synthesis:DefaultSynthesisLinearFunction",
+            "linear_function.depth_opt = test_stuff:LinearFunctionSynthesisPluginForDepth",
+            "linear_function.count_opt = test_stuff:LinearFunctionSynthesisPluginForCount",
```

Comment:
```suggestion
```
I don't think these have been merged to terra yet. Maybe this was left over from testing?

Reply:
Oops, thanks for noticing this, indeed these do not belong there.